dashboards/azure-app-gateway/azure-app-gateway.json

```diff
@@ -25,7 +25,7 @@
             "nrqlQueries": [
               {
                 "accountIds": [],
-                "query": "FROM Metric SELECT SUM(`azure.network.applicationgateways.TotalRequests`) AS 'Total Requests', SUM(`azure.network.applicationgateways.FailedRequests`) AS 'Total Failed Requests', SUM(`azure.network.applicationgateways.BlockedReqCount`) AS 'Total Blocked Requests', SUM(`azure.network.applicationgateways.CurrentConnections`) AS 'Total Current Connections', SUM(`azure.network.applicationgateways.HealthyHostCount`) AS 'Total Healthy Backend Hosts', SUM(`azure.network.applicationgateways.UnhealthyHostCount`) AS 'Total Unhealthy Backend Hosts'"
+                "query": "SELECT SUM(`totalRequests.Total`) AS `Total Requests`, SUM(`failedRequests.Total`) AS `Total Failed Requests`, SUM(`blockedReqCount.Total`) AS `Total Blocked Requests`, SUM(`currentConnections.Total`) AS `Total Current Connections`, SUM(`healthyHostCount.Total`) AS `Total Healthy Backend Hosts`, SUM(`unhealthyHostCount.Total`) AS `Total Unhealthy Backend Hosts` FROM AzureApplicationGatewaySample WHERE (`provider` = 'AzureApplicationGateway')"
```

Comment:
Its common to all the customers and it is not expected to change.

Reply:
This dashboard is transformed to Samples because the app-gateway integration uses Polling to gather the data. In Samples provider is specified to correctly show the data.